src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -1448,8 +1461,23 @@ public ServiceConfig getFilteredConfig() {
   }
 
   @Override
-  public void setConfig(ServiceConfig config) {
+  public ServiceConfig setConfig(ServiceConfig config) {
```

Comment:
normally setters don't return anything.  why change this signature?

Reply:
I'll change it back, was thinking the echo would be useful ..
like fluent style - https://medium.com/miro-engineering/fluent-setter-breaking-the-convention-33ce3433126e

Reply:
set to void